src/util/Version.java.template

```diff
@@ -0,0 +1,12 @@
+package util;
+
+import picocli.CommandLine;
+
+public class Version implements CommandLine.IVersionProvider {
+    public static final String version = "%%VERSION%%";
+
+    @Override
+    public String[] getVersion() {
+        return new String[] {"1.0.0." + version};
```

Comment:
I've made a release for before these changes to be 1.0.  Could make this 1.1 or 2.0 to reflect updated interface.

Reply:
Sure ;)

Reply:
Sure